powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/ActivationHistoryRequest.java

```diff
@@ -30,8 +32,14 @@
 @Data
 public class ActivationHistoryRequest {
 
+    @Schema(description = "Activation identifier")
+    @NotBlank(message = "Activation ID must not be empty when requesting activation history")
     private String activationId;
+
+    @Schema(description = "Timestamp from used for timestamp filter")
     private Date timestampFrom;
+
+    @Schema(description = "Timestamp to used for timestamp filter")
```

Comment:
Suggesting different wording.
```suggestion
    @Schema(description = "The ending timestamp for filtering activation history")
```